test/system-tests/detect_package_manager_test.sh

```diff
@@ -39,6 +39,15 @@ function test_detect_package_manager_on_open_suse_systems_with_zypper()
 	assertEquals "did not detect zypper" "zypper" "$package_manager"
 }
 
+function test_detect_package_manager_on_void_systems_with_xbps()
+{
+	command -v lsb_release >/dev/null &&
+		$(lsb_release -c|grep -q void) &&
```

Comment:
Is this line really necessary? Seems like testing for the `xbps-install` command would be enough?